src/main/java/seedu/address/ui/reservation/ReservationCard.java

```diff
@@ -58,8 +60,9 @@ public ReservationCard(Reservation reservation, int displayedIndex) {
         this.reservation = reservation;
         id.setText(displayedIndex + ". ");
         name.setText(reservation.getName().toString());
-        pax.setText(reservation.getPax().value);
-        dateTime.setText(reservation.getDateTime().toString());
+        pax.setText("Pax: " + reservation.getPax().value);
```

Comment:
Can consider changing to:

```suggestion
        pax.setText("Pax: " + reservation.getPax().toString());
```

since you're setting text to a label.

Reply:
the "value" attribute of Pax is already a String

Reply:
In response to your reply below, then probably can just call `.getPax().toString()` directly. 
Also I'm wondering why is `value` a String though? Wouldn't the "number of persons" be represented by an `int`?

Reply:
I was following the existing implementation of the `Phone` class, since it stored the actual phone number value as a `String`.

I decided to follow suit in case using `int` would screw up the implementation

Reply:
Ok